src/main/java/com/iota/iri/network/TipRequester.java

```diff
@@ -0,0 +1,106 @@
+package com.iota.iri.network;
+
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.network.neighbor.Neighbor;
+import com.iota.iri.network.pipeline.TransactionProcessingPipeline;
+import com.iota.iri.service.milestone.LatestMilestoneTracker;
+import com.iota.iri.storage.Tangle;
+import com.iota.iri.utils.thread.ThreadIdentifier;
+import com.iota.iri.utils.thread.ThreadUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * The {@link TipRequester} requests tips from all neighbors in a given interval.
+ */
+public class TipRequester {
```

Comment:
Can you make a simple interface for this,
like with `TransactionRequesterWorker`?

Reply:
Done.